.github/actions/setup-dependencies/action.yaml

```
@@ -42,6 +42,14 @@ runs:
           protoc --version
           rm -rf protoc.zip
           echo "::endgroup::"
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)